release/models/sampling/openconfig-sampling-sflow.yang

```diff
@@ -148,6 +148,17 @@ module openconfig-sampling-sflow {
         to sFlow collectors.";
     }
 
+    leaf datagram-size {
+      type uint16;
+      units bytes;
+      default 1400;
```

Comment:
I'd suggest removing the default value.
- I don't think there's a common default across the industry, or a good reason to enforce it as 1400
- even the arista reference that mentions this value states that `Only software sFlow supports this feature.`

Reply:
:-) I see, thanks.